R/link_delayed_discharge_eps.R

```diff
@@ -50,7 +50,7 @@ link_delayed_discharge_eps <- function(data, year) {
     )
 
   by_dd <- dplyr::join_by(
-    chi,
+    "chi",
```

Comment:
Does this definitely work?

I broke it previously by doing something very similar 😅